sites/public/styles/overrides.scss

```diff
@@ -164,4 +164,31 @@
       --button-border-radius-md: var(--seeds-rounded-3xl);
     }
   }
+
+  // The multi-image card overrides below exist outside of the jurisdictional class. This allows it to still get the lakeview overrides - if the class does exist, then apply the same overrides to its sibling.
+  .lakeview + div {
+    .image-card__gallery-modal {
+      @extend .lakeview;
+    }
+  }
```

Comment:
question: Will we have to do this for all of our jurisdictional repos? Or is this just the case because we are adding lakeview overrides via a class?

Reply:
This is just for the lakeview class, it works in Detroit without this